go.mod

```diff
@@ -4,49 +4,44 @@ go 1.19
 
 require (
 	cosmossdk.io/math v1.0.0-beta.4
-	github.com/CosmWasm/wasmd v0.29.2
+	github.com/CosmWasm/wasmd v0.30.0
```

Comment:
this change will have no impact as we `replace` this below. You'll need to check on notionals v46- wasm0.30 branch

Reply:
reverted

Reply:
https://github.com/ingenuity-build/quicksilver/pull/346/commits/dd952983ea7fa77e9c453aa3df3ee339c04e7263